pages/interview-process.md

```diff
@@ -11,54 +15,23 @@ The first step in the interview process is the phone screen, a 30-minute phone c
 
 After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews. Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday. 
 
-### Basic rubric and criteria
-
-Interviewers evaluate candidates using the basic rubric below, without discussion or input from interview partners or other team members. After they complete a written evaluation, all interviewers meet and discuss the candidate. 
-
-- Principles alignment
-
-	 - Strong desire to serve
-	 - Embraces motivations of open source 
-	 - Embraces user feedback as a way of driving a project 
-	 - Understands working iteratively or as agile team 	
-
-- Technical competence
-
-	 - Knowledge of programming language(s) 
-	 - Knowledge of web framework(s) or other relevant frameworks
-	 - Demonstrated ability to write well-structured code
-	 - Understanding of web technology 
-	 - Understanding of modularity
-
--  Communication and collaboration skills
-
-	 - Ability to articulate technical concepts 
-	 - Ability to speak concisely and boldly
-	 - Understands working iteratively or as agile team
-	 - Teachability
-	 - Quick learning
-	 - Flexibility of thought
-
-- Leadership and entrepreneurship
-
-	 - Capable of evangelism
-	 - Can they teach us something we don’t know?
-	 - Will they “raise the bar” of our team?
-
--  Creativity
-
-	 - Ability to translate creativity into practical solutions 
-	 - Change hearts and minds
-
-### Interviewing for a specific team? 
-
-Check out more interviewing information for each of our teams. 
-
-<ul>
-	{% for interview in site.interviews %}
```

Comment:
@rooneywp Can we keep these team-specific ones? I appreciate that this is trying to address #105, but the engineering rubric, for instance, includes a lot of valuable specifics that will otherwise be lost here.